webapp/models.py

```diff
@@ -153,6 +153,35 @@ def notice_type(self):
 
         return ""
 
+    @hybrid_property
+    def package_list(self):
+        if not self.release_packages:
+            return []
+
+        package_list = []
+        for codename, packages in self.release_packages.items():
+            for package in packages:
+                package_list.append(package["name"])
+
+        return set(sorted(package_list))
+
+    @hybrid_property
+    def related_notices(self):
+        related_notices_ids = [self.id]
```

Comment:
Why is `self` in related?

Reply:
Do you think we could call it `seen_notice_ids` or something? To help hint that it's just a utility list for the loop?